packages/stable/src/types.ts

```diff
@@ -1990,6 +1990,16 @@ export interface Revision3D {
    * The rotation is expressed by Euler angles in radians and in XYZ order.
    */
   rotation?: Tuple3<number>;
+  /**
+   * Global translation to be applied to the entire model.
+   * The translation is expressed in meters.
+   */
+  translation?: Tuple3<number>;
+  /**
+   * Global scale to be applied to the entire model.
+   * The scale is expressed as scale along the X, Y and Z axes.
+   */
+  scale?: Tuple3<number>;
```

Comment:
Just out of curiousity: is there any reason we have an alias for an array? Is there some type constraint somewhere in the SDK or is it just a hint for users?

Reply:
I just took the same type as `rotation`, so I didn't think about it :sweat_smile: But I expected it to be defined as `[T, T, T]`, and not `T[]`. Not sure why not :thinking: